rfd/0001-testing-guidelines.md

```diff
@@ -117,21 +117,21 @@ Subtests
 ```go
 func TestParseInt(t *testing.T) {
 	tests := []struct {
-		desc    string
-		in      string
-		want    int
-		wantErr bool
+		desc      string
+		in        string
+		want      int
+		assertErr assert.ErrorAssertionFunc
 	}{
-		{desc: "positive", in: "123", want: 123},
-		{desc: "negative", in: "-123", want: -123},
-		{desc: "non-numeric", in: "abc", wantErr: true},
-		{desc: "empty", in: "", wantErr: true},
+		{desc: "positive", in: "123", want: 123, assertErr: assert.NoError},
+		{desc: "negative", in: "-123", want: -123, assertErr: assert.NoError},
+		{desc: "non-numeric", in: "abc", assertErr: assert.Error},
+		{desc: "empty", in: "", assertErr: assert.Error},
 	}
 	for _, tt := range tests {
 		t.Run(tt.desc, func(t *testing.T) {
 			got, gotErr := parseInt(tt.in)
-			assert.Equal(err != nil, tt.wantErr, "got error:", err)
-			assert.Equal(got, tt.want)
+            tt.assertErr(t, err)
```

Comment:
spacing, it looks like this:
![image](https://user-images.githubusercontent.com/43280172/95919234-137f5480-0d62-11eb-8402-ec04f9309923.png)

Reply:
also, should it be `tt.assertErr(t, gotErr)`? or maybe `gotErr` to `err`

Reply:
fixed